lib/wings/model_transformer.rb

```diff
@@ -212,7 +212,7 @@ def self.run(obj, keys)
 
       def self.attribute_ids_for(name:, obj:)
         attribute_value = obj.try(name)
-        return unless attribute_value.present?
+        return if attribute_value.nil?
```

Comment:
For a File, when attribute_value.content is an empty string, attribute_value.present? returns false even though attribute_value is_a ActiveFedora::File object.  Switching to check for nil catches this and should still appropriately fail the test obj.try returns nil.

Reply:
Added requested tests.